sycl/source/detail/pi_opencl.cpp

```diff
@@ -159,6 +159,33 @@ pi_program OCL(piProgramCreate)(pi_context context, const void *il,
   return pi_cast<pi_program>(resProgram);
 }
 
+pi_sampler OCL(piSamplerCreate)(pi_context context,
+                                const cl_sampler_properties *sampler_properties,
+                                pi_result *error_code) {
+  cl_sampler returnSampler;
+  cl_bool normalizedCoords;
+  cl_addressing_mode addressingMode;
+  cl_filter_mode filterMode;
```

Comment:
Should I initialize these parameters some default values?

Reply:
either this, or make sure they are all initialized below